app/templates/components/running-jobs.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if this.jobs.isLoaded}}
-  {{#if this.jobs.length}}
-    {{#each this.jobs as |job|}}
+  {{#if this.runningJobs.length}}
```

Comment:
Same here with `{{#each}}{{else}}{{/each}}`. I understand that it was here before, but it's easy to improve ;)